### PR TITLE
Refactor carousel side arrows styling and spacing in content

### LINKED ISSUE #607: Side Arrows on Testimonials ,Events Winners
### What would you like to share?

Screenshot 2023-05-14 133807
side arrows are in top of the cards.
I want to fix this issue

### Additional information

also it won't support any gestues whil slide from left to right or vice versa. I can fix it but how can i contribute? 
I need some help with this.Thank you

### PR DESCRIPTION
**Pull request by nayan-dey**
I refactored the styling and adjusted the spacing of the side arrows in the carousel component. The changes made improve the visual appearance and ensure a more consistent user experience.

## Fixes Issue

This PR fixes the following issues:

closes #607 

Previously, the side arrows in the carousel had a default color and spacing that didn't align with the overall design aesthetic of the content. Recognizing the importance of consistency and cohesiveness in user interfaces, this commit addresses these concerns and brings significant improvements to the component's appearance.

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested **changes.**

## Screenshots

Add all the screenshots which support your changes

### **carousal before changes**

arrow1

### **made some changes on the color and spacing**

arrow2
arrow3
arrow4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "react": "^18.2.0",
     "react-countup": "^6.4.1",
     "react-dom": "^18.2.0",
-    "react-icons": "^4.6.0",
+    "react-icons": "^4.8.0",
```
Comment (Chandraprakash-Darji) [inline package.json:27]: Don't update deps

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nayan-dey) [reply package.json:27]: Can I undo this part or it would be okay for this time

Comment (Chandraprakash-Darji) [reply package.json:27]: just undo it

Comment (nayan-dey) [reply package.json]: Hey, I did what you said.

**Review by Chandraprakash-Darji — APPROVED** [review]

**Review by Chandraprakash-Darji — APPROVED** [review]

**Pull request merged**